### PR TITLE
CSS and script

### PR DESCRIPTION
**Pull request by rkraeher**
adding empty js and initial css file

### CODE REVIEW

**Comment by Nolanws1** [comment]
Okay!

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.body {
```
Comment (Nolanws1) [inline css/style.css:1]: Change to DARK BLUE

Comment (Nolanws1) [reply css/style.css:1]: No

**Comment by mitch-t** [comment]
Good

**Review by jcquper — APPROVED** [review]

**Pull request merged**